src/test/java/seedu/address/testutil/TypicalTeams.java

```diff
@@ -0,0 +1,65 @@
+package seedu.address.testutil;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import seedu.address.AlfredException;
+import seedu.address.model.entity.Id;
+import seedu.address.model.entity.Location;
+import seedu.address.model.entity.Name;
+import seedu.address.model.entity.PrefixType;
+import seedu.address.model.entity.ProjectType;
+import seedu.address.model.entity.Score;
+import seedu.address.model.entity.SubjectName;
+import seedu.address.model.entity.Team;
+import seedu.address.model.entitylist.TeamList;
+
+/**
+ * A utility class containing a list of {@code Team} objects to be used in tests.
+ */
+public class TypicalTeams {
```

Comment:
Might have a problem here with static variables. Because let's say we mutate the arttributes of them in 1 test, they will be affected in the next test because both point to the same variable.  Will need a reset test. I wrote one in my PR